src/univers/swift_version_range.py

```diff
@@ -0,0 +1,57 @@
+import attr
+
+from univers.version_constraint import VersionConstraint
+from univers.version_range import VersionRange
+from univers.versions import SwiftVersion
+
+
+@attr.s(auto_attribs=True, frozen=True)
+class SwiftVersionRange(VersionRange):
+    expression: str
+    scheme: str = "swift"
+    version_class: type = SwiftVersion
+    constraints: list = attr.ib(init=False)
+
+    def __attrs_post_init__(self):
+        object.__setattr__(self, "constraints", self.parse(self.expression))

```

Comment:
Why is this needed? Aren’t we already taking care of it in the base `VersionRange` class?